concent_api/core/tests/test_payments_database_transaction_storage.py

```diff
@@ -6,14 +8,19 @@
 from core.payments.storage import DatabaseTransactionsStorage
 
 
+def _sign(tx: Transaction) -> None:
+    tx.sign(os.urandom(32))
+
+
 class DatabaseTransactionsStorageTest(TestCase):
 
     multi_db = True
 
     def setUp(self):
         super().setUp()
 
-        self.storage = DatabaseTransactionsStorage(5)
+        self.storage = DatabaseTransactionsStorage()
+        self.storage.init(5)
```

Comment:
please extract to variable/constant (i. e. `self.initial_nounce`) and reuse it throughout the tests.